AUTHORS.md

```diff
@@ -18,18 +18,22 @@
 - Jean-Luc Vay (LBNL)
 - David P. Grote (LLNL)
 - Ivo F. Sbalzarini (TU Dresden, MPI-CBG)
-- Stephan Kuschel (IOQ Jena, now SLAC)
-- Michael Bussmann (HZDR)
+- Stephan Kuschel (IOQ Jena, now Uni Hamburg)
+- Michael Bussmann (CASUS, HZDR)
 
 
-### Reviewers and Substantive Contributions in openPMD 2.0 (2017-2020)
+### Reviewers and Substantive Contributions in openPMD 2.0 (2017-2021)
 
 Above authors and:
 
 - David Sagan (Cornell)
 - Christopher Mayes (SLAC)
 - Frédéric Pérez (LULI)
 - Fabian Koller (HZDR)
+- Franz Poeschel (CASUS)
+- Carsten Fortmann-Grote (MPI-EvolBio)
```

Comment:
My affiliations should read EU XFEL, MPI-EvolBio

Reply:
Thank you Carsten, I updated your affiliations right away.